src/algorithms/reco/TransformBreitFrame.h

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 John Lajoie
+
+#pragma once
+
+#include <edm4hep/MCParticleCollection.h>
+#include <edm4eic/InclusiveKinematicsCollection.h>
+#include <edm4eic/ReconstructedParticleCollection.h>
+#include <spdlog/logger.h>
+#include <map>
+#include <memory>
+#include <string>
+
+namespace eicrecon {
+
+  class TransformBreitFrame {
+
+    public:
```

Comment:
@veprbl can correct if I'm wrong here, but we'll probably want to use the algorithms interface. So this part will look something like so:

```suggestion
 using TransformBreitFrameAlgorithm = algorithms::Algorithm<
   algorithms::Input<
     edm4hep::MCParticleCollection*,
     edm4eic::InclusiveKinematicsCollection*,
     edm4eic::ReconstructedParticleCollection*
   >,
   algorithms::Output<
     std::unique_ptr<edm4eic::ReconstructedParticleCollection>
   >
  >;
  
  class TransformBreitFrame : 
    public TransformBreitFrameAlgorithm,
    public WithPodConfig<NoConfig> {

    public:
```